model/dce-stdio.cc

```diff
@@ -342,27 +299,19 @@ int dce_fclose_unconditional (FILE *file)
   // Note: it is important here not to call the Current function here
   // because we need to be able to run this function even if there is no context.
   // For example, this is why we have no call to NS_LOG_FUNCTION (Current () ...);
-  struct my_IO_FILE_plus *fp = (struct my_IO_FILE_plus *)file;
-  static struct my_IO_jump_t vtable;
-  memcpy (&vtable, fp->vtable, sizeof(struct my_IO_jump_t));
-  vtable.__close = (void*)my_close_unconditional;
-  vtable.__write = (void*)my_write_unconditional;
-  vtable.__seek = (void*)my_seek_unconditional;
-  fp->vtable = &vtable;
-  fclose (file);
+  // TODO
+  //fclose(file);
+  // Maybee dce_close(fileno(file)) ?
```

Comment:
```dce_fclose_unconditional()``` is always done as a cleanup step under ```DceManager::DeleteProcess()``` which is always outside of simulation context, thus doing an ```fclose``` here would lead to a SIGSEGV on the m_context. 
But, also is it a good practise in terms of memory management to leave FILE pointers open before stopping the process ?